app/jobs/cache_analytics_job.rb

```diff
@@ -37,72 +46,40 @@ def perform(*_args)
-    counts.concat(monitoree_counts_by_exposure_country(analytic_id, monitorees, false))
+    counts.concat(monitoree_counts_by_age_group(analytic_id, monitorees))
+    counts.concat(monitoree_counts_by_sex(analytic_id, monitorees))
+    counts.concat(monitoree_counts_by_ethnicity(analytic_id, monitorees))
+    counts.concat(monitoree_counts_by_race(analytic_id, monitorees))
+    # counts.concat(monitoree_counts_by_sexual_orientation(analytic_id, monitorees))
+    counts.concat(monitoree_counts_by_reporting_method(analytic_id, monitorees))
+    counts.concat(monitoree_counts_by_risk_factor(analytic_id, monitorees))
+    counts.concat(monitoree_counts_by_exposure_country(analytic_id, monitorees))
 
     # Active and overall counts for date of last exposure
-    counts.concat(monitoree_counts_by_last_exposure_date(analytic_id, monitorees, true))
-    counts.concat(monitoree_counts_by_last_exposure_date(analytic_id, monitorees, false))
-    counts.concat(monitoree_counts_by_last_exposure_week(analytic_id, monitorees, true))
-    counts.concat(monitoree_counts_by_last_exposure_week(analytic_id, monitorees, false))
-    counts.concat(monitoree_counts_by_last_exposure_month(analytic_id, monitorees, true))
-    counts.concat(monitoree_counts_by_last_exposure_month(analytic_id, monitorees, false))
-
-    counts
-  end
-
-  # Total monitoree counts
-  def self.monitoree_counts_by_total(analytic_id, monitorees, active_monitoring)
-    monitorees.monitoring_active(active_monitoring)
-              .group(:exposure_risk_assessment)
-              .order(:exposure_risk_assessment)
-              .size
-              .map do |risk_level, total|
-                monitoree_count(analytic_id, active_monitoring, 'Overall Total', 'Total', risk_level, total)
-              end
-  end
-
-  # Monitoree counts by monitoring status (symptomatic, non-reporting, asymptomatic)
-  def self.monitoree_counts_by_monitoring_status(analytic_id, monitorees)
-    counts = []
-    MONITORING_STATUSES.each do |monitoring_status|
-      monitorees.monitoring_status(monitoring_status)
-                .group(:exposure_risk_assessment)
-                .order(:exposure_risk_assessment)
-                .size
-                .each do |risk_level, total|
-                  counts.append(monitoree_count(analytic_id, true, 'Monitoring Status', monitoring_status, risk_level, total))
-                end
-    end
+    counts.concat(monitoree_counts_by_last_exposure_date(analytic_id, monitorees))
+    counts.concat(monitoree_counts_by_last_exposure_week(analytic_id, monitorees))
+    counts.concat(monitoree_counts_by_last_exposure_month(analytic_id, monitorees))
     counts
   end
 
   # Monitoree counts by age group
-  def self.monitoree_counts_by_age_group(analytic_id, monitorees, active_monitoring)
+  def self.monitoree_counts_by_age_group(analytic_id, monitorees)
+    counts = []
+    # Some jurisdictions are using `1-1-1900` as a "fake birthdate" where data might be invalid or unknown
```

Comment:
this is more of a general comment rather than one specific for this PR, but should we then consider maybe not making `date_of_birth` a required field in enrollment?